### PR TITLE
Submitting this to see if I am doing it correctly

### PR DESCRIPTION
**Pull request by tdegeorge**

### CODE REVIEW

**Comment by siwelwerd** [comment]
Marking this as draft since you are still working, and noting that the build is failing on this commit.

```
critical: Opening and ending tag mismatch: statement line 69 and p, line 90, column 9
------------------------
critical: Failed to build.  Exiting...
```

File: source/precalculus/source/06-TR/03.ptx [hunk 1]
```diff
@@ -12,6 +12,97 @@
...
+          The <term>opposite side</term> is across from a given angle.
+        </p>
+      </li>
+      <li>
+        <p>
+          The <term>adjacent side</term> is the non-hypotenuse side that is next to a given angle.
+        </p>
+      </li>
+    </ul>
+    </p>
```
Comment (siwelwerd) [inline source/precalculus/source/06-TR/03.ptx:90]:
```suggestion

```

You have an extraneous `</p>` here that was causing the error.

File: source/precalculus/source/06-TR/04.ptx [hunk 2]
```diff
@@ -12,6 +12,135 @@
...
+            </ul>
+          </p>
+        </statement>
+      </definition>
+
+      <activity xml:id="exploring-45-45-90-triangle">
+        <introduction>
+          <p>
+            In this activity, let's explore the relationship of the <m>45-45-90</m> special right triangle.
+        </introduction>
```
Comment (siwelwerd) [inline source/precalculus/source/06-TR/04.ptx:45]:
```suggestion
        <introduction>
          <p>
            In this activity, let's explore the relationship of the <m>45-45-90</m> special right triangle.
          </p>
        </introduction>
```

If you look at the PreTeXt output, it says
```
critical: Failed to build.  Exiting...
critical: Opening and ending tag mismatch: p line 43 and introduction, line 45, column 24
```

Which is telling us that the `<p>` opened on line 43 is not being closed before the `</introduction>` tag on line 45.  So commit this suggestion and this error should be fixed.

File: source/precalculus/source/06-TR/03.ptx [hunk 3]
```diff
@@ -12,6 +12,611 @@
...
+<remark>
+  <p>
+    In this section, we will learn how to use right triangles to evaluate trigonometric functions. Before doing that, however, let's review some key concepts of right triangles that can also come in handy when solving.
+  </p>
+</remark>
+
+<definition xml:id="def-pythagorean-theorem">
+  <statement>
+    <p>
+      The <term>Pythagorean Theorem</term> is <me>a^2+b^2=c^2</me>, where <m>a</m> and <m>b</m> are lengths of the legs of a right triangle and <m>c</m> is the length of the hypotenuse. If we know the lengths of any two sides of a right triangle, we can find the length of the third side.
```
Comment (StevenClontz) [inline source/precalculus/source/06-TR/03.ptx:24]:
```suggestion
      The <term>Pythagorean theorem</term> is <me>a^2+b^2=c^2</me>, where <m>a</m> and <m>b</m> are lengths of the legs of a right triangle and <m>c</m> is the length of the hypotenuse. If we know the lengths of any two sides of a right triangle, we can find the length of the third side. 
```

Comment (StevenClontz) [reply source/precalculus/source/06-TR/03.ptx:24]: Actually, capitalized is better.

**Comment by siwelwerd** [comment]
This was pre-empted by #365

**Pull request closed without merging**